Makefile

```diff
@@ -18,17 +18,20 @@ REGISTRY ?= andyzhangx
 REGISTRY_NAME = $(shell echo $(REGISTRY) | sed "s/.azurecr.io//g")
 IMAGE_NAME ?= smb-csi
 IMAGE_VERSION ?= v0.7.0
+WIN_IMAGE_VERSION ?= v0.6.0-windows-1809-amd64

```

Comment:
should be `v0.7.0-windows-1809-amd64` which is `${IMAGE_VERSION}-windows-1809-amd64` ?

Reply:
Done. Thanks!